cmd/csi-sanity/sanity_test.go

```diff
@@ -45,7 +45,8 @@ func TestSanity(t *testing.T) {
 		return
 	}
 	if len(endpoint) == 0 {
-		t.Fatalf("--%s.endpoint must be provided with an CSI endpoint", prefix)
+		t.Fatalf("--%sendpoint must be provided with an CSI endpoint", prefix)
 	}
+	fmt.Printf("Endpoint = %s\n", endpoint)
```

Comment:
Do not use fmt. This is more like a unit test. Use this instead if you want: http://onsi.github.io/ginkgo/#logging-output .


I feel it is not really necessary to print out the endpoint, but I wouldn't be against only printing it on verbose output.

Reply:
Thanks. I will remote Line 50